internal/engine/upper.go

```diff
@@ -196,6 +200,9 @@ var UpperReducer = store.Reducer(func(ctx context.Context, state *store.EngineSt
 		handleAnalyticsNudgeSurfacedAction(ctx, state)
 	case UIDUpdateAction:
 		handleUIDUpdateAction(state, action)
+	case store.LogEvent:
+		// handled as a LogAction, do nothing
```

Comment:
Why not use a `fallthrough` instead of what happens on L142?

Reply:
hmmmm...that wouldn't work. we never want to fallthrough to a subsequent case.